File: Easy/JungleCamping.py
noises = input().split()
for noise in noises:
    if noise == 'Grr':
        print('Lion', end=' ')
    elif noise == 'Rawr':
        print('Tiger', end=' ')
    elif noise == 'Ssss':
        print('Snake', end=' ')
    else:
        print('Bird', end=' ')

File: Easy/MyC#Solutions/JungleCamping.cs
using System;

class Program {
    static void Main(string[] args) {
        string[] noises = Console.ReadLine().Split(" ");
        foreach (var noise in noises){
            if (noise == "Grr")
                Console.Write("Lion ");
            else if (noise == "Rawr")
                Console.Write("Tiger ");
            else if (noise == "Ssss")
                Console.Write("Snake ");
            else
                Console.Write("Bird ");
        } 
    }
}